corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -37,6 +37,7 @@
   {% initial_page_data 'edit_context' edit_context %}
   {% initial_page_data 'edit_formplayer' edit_formplayer %}
   {% initial_page_data 'mapbox_access_token' mapbox_access_token %}
+  {% initial_page_data 'default_geocoder_location' default_geocoder_location %}
```

Comment:
Also this, is this needed in reports?

Reply:
(same comment as above, this is edit forms)